lib/acts_as_taggable_on_mongoid/taggable/tag_type_definition/changeable.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+module ActsAsTaggableOnMongoid
+  module Taggable
+    class TagTypeDefinition
+      module Changeable
```

Comment:
`Changeable` is the Mongoid module that does the same basic thing for fields, so I kept the name.